sql/updates/2.3.0--2.3.1.sql

```diff
@@ -0,0 +1 @@
+DROP FUNCTION IF EXISTS _timescaledb_internal.refresh_continuous_aggregate;
```

Comment:
This function is dropped because it moved to the experimental schema right? But I didn't see we are including the experimental schema in this release, so is this correct?

I guess dropping the function doesn't hurt, though, since they are recreated during the upgrade if I am not wrong.

Reply:
No, the function is dropped because it is re-defined. We do not introduce the experimental schema in this release since it causes issues with the update tests and we need to go over it separately.